restaking_sdk/src/sdk.rs

```diff
@@ -311,6 +312,25 @@ pub fn operator_set_secondary_admin(
     }
 }
 
+pub fn operator_set_fee(
```

Comment:
can you add config here? just in case we set cap/change global behavior through it

Reply:
done